src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
 // write your code here
+const table = document.querySelector('table');
+const rows = table.rows;
+
+for (const row of rows) {
+  const copyOfColumn = [];
+
+  copyOfColumn.push(row.cells[1]);
```

Comment:
The `copyOfColumn` array is being used to store the second column cell of each row, but it is not necessary to store it in an array since you are only dealing with one cell per iteration. You can directly use `row.cells[1]` in the insertion logic.